bundle/internal/tf/schema/resource_app.go

```diff
@@ -0,0 +1,102 @@
+// Generated from Databricks Terraform provider schema. DO NOT EDIT.
```

Comment:
All of the TF changes regenerated based on local TF override with this branch https://github.com/databricks/terraform-provider-databricks/pull/4099

Will be updated when the change is released in TF provider and provider updated in CLI